boot/setup.sh

```diff
@@ -133,6 +133,10 @@ function manta_setup_nginx {
         '/var/log/mako-{access,error}.log'
 }
 
+function manta_setup_garbage_deleter {
+    svccfg import /opt/smartdc/mako/smf/manifests/garbage-deleter.xml
+}
```

Comment:
Should there be a `manta_add_logadm_entry "garbage-deleter"`?  Or don't want hourly rotation and upload?

Reply:
Probably should be. I don't use it in my environment but we should match what others zones do for sure. Thanks for catching this.

Reply:
Done with latest commit.